cupy/cuda/nccl.pxd

```diff
@@ -14,3 +14,18 @@ cpdef enum:
     NCCL_DOUBLE = 4
     NCCL_INT64 = 5
     NCCL_UINT64 = 6
+
+    NCCL2_INT8 = 0
```

Comment:
To solve it in a PR to support NCCL2 for chainermn, NCCL_FLOAT of nccl version 2 is converted to one of nccl version 1.
https://github.com/chainer/chainermn/pull/105/files